### DISCUSSION TITLE
Battery warnings

### DISCUSSION THREAD

**Discussion by Blooskyy** [EdgeTX/edgetx#1984] [General]
A question regarding battery warnings. 
I have created custom battery warnings by using logical switches and special functions, and this works perfectly. 
However in addition to these warnings the default radio battery warning is also given. Is there a way to disable the default warnings as these aren't required? 

Do Betaflight power settings have any effect on the TX warnings?    
Thanks

Latest EdgeTX version
TX16S MKII

**Comment by Blooskyy** [comment]
Solved - These were iNav telemetry LUA script warnings, and deleting the script has resolved the issue.

**Reply by pfeerick** [reply]
I believe you can change or turn off the battery warnings in the inav telemetry configuration screen... https://luatelemetry.readthedocs.io/en/latest/Configuration-Settings/
